### ISSUE TITLE
Tries collection twice when aborted during it

### ISSUE THREAD

**Issue by blueyed** [joeyespo/pytest-watch#52]
When aborting ptw in the collection phase it will not quit, but retry it:

```
% ptw ...
^CError: Could not run --collect-only to find the pytest config file. Trying again without silencing stdout..
```

I think it should exit on SIGINT here.

**Comment by joeyespo** [comment]
Ooh, nice find! Looks like `KeyboardInterrupt` was never handled in the new config collecting code.

Fixing soon.

**Comment by joeyespo** [comment]
Just published v4.1.0 with this fix. Thanks, @blueyed!
